tests/auth_tests/test_validators.py

```diff
@@ -259,3 +264,71 @@ def test_ascii_validator(self):
             with self.subTest(invalid=invalid):
                 with self.assertRaises(ValidationError):
                     v(invalid)
+
+    def test_username_minimum_length_validator(self):
+        valid_usernames = ['guido', 'raymond']
+        invalid_usernames = ['1t', 'tw', '']
+        v = validators.UsernameMinimumLengthValidator(min_length=3)
+        for valid in valid_usernames:
+            with self.subTest(valid=valid):
+                v(valid)
+        msg = 'This username is too short. It must contain at least 3 characters.'
+        for invalid in invalid_usernames:
+            with self.subTest(invalid=invalid):
+                with self.assertRaisesMessage(ValidationError, msg):
+                    v(invalid)
+
+    def test_username_validators_help_text(self):
+        validator_config = [
+            {'NAME': 'django.contrib.auth.validators.UnicodeUsernameValidator'},
+            {'NAME': 'django.contrib.auth.validators.UsernameMinimumLengthValidator', 'OPTIONS': {
+                'min_length': 3,
+            }},
+        ]
+        ASCII_and_unicode = _(
```

Comment:
I would ensure this variable name begins with a lowercase letter somehow, like`unicode_help_text`